helm-charts/common/speecht5/templates/deployment.yaml

```diff
@@ -54,6 +53,9 @@ spec:
           command: ['sh', '-c']
           args:
             - |
+              # NOTE: temporary workaround
+              # https://github.com/huggingface/huggingface_hub/issues/2708
+              pip install 'huggingface-hub==0.26.5';
               huggingface-cli download --cache-dir /data --token $(HF_TOKEN)  {{ .Values.TTS_MODEL_PATH | quote }};
               huggingface-cli download --cache-dir /data --token $(HF_TOKEN)  {{ .Values.VOCODER_MODEL| quote }};
```

Comment:
Documentation for the older version of downloader:
https://github.com/huggingface/huggingface_hub/tree/v0.17-release#login

Tells to use:
`huggingface-cli login --token $(HF_TOKEN)`

What if that is used instead upgrading `huggingface-hub` & giving tokens to `download` commands?

Reply:
`login` command works